<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="origianl_img_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="output_img_lbl">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="OutPut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="sobel_rb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sobel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="gray_scale_rb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gray Scale</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="binary_rb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Binary - BW</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="histo_rb">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogram</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="histo_rb_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogram</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="histo_rb_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Histogram</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="normalize_filter_rb">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Normalized Block 
Filter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="gaussian_rb">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gaussian Filter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="median_rb">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Median Filte</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sobel_gb">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Sobel Operator Parameter</string>
    </property>
    <widget class="QSlider" name="scale_hsb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="delta_hsb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="scale_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale:</string>
     </property>
    </widget>
    <widget class="QLabel" name="delta_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Delta:</string>
     </property>
    </widget>
    <widget class="QLabel" name="scale_value_lbl">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="delta_value_lbl">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="binary_gb">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Binary Parameter</string>
    </property>
    <widget class="QSlider" name="threshold_hsb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="threshold_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
    <widget class="QLabel" name="threshold_value_lbl">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>128</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
